src/iframe-helper.js

```diff
@@ -125,15 +126,9 @@ function getListenForEvents(parentWin, sentinel, origin, triggerWin) {
     const contentWindow = we.frame.contentWindow;
     if (!contentWindow) {
       setTimeout(dropListenSentinel, 0, listenSentinel);
-    } else if (sentinel === 'amp') {
```

Comment:
so, we end up removing this whole block for "sentinel==='amp''.
why was it affecting the A4A iframes? I don't think they have sentinel of 'amp', do they?

will this break our non-3p iframes which has sentinel = 'amp'?